### PR TITLE
Add Bitcoin command

### PR DESCRIPTION
**Pull request by itsrishub**
Added a command where users can check the price of current BTC is USD using an API with no rate limit.

### CODE REVIEW

**Comment by LevaniVashadze** [comment]
Could you transfer this command to Info.py and delete btc.py

File: cogs/btc.py [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+
+
+class BTCCommand(commands.Cog, name="BTC"):
+    """BTC Command"""
+
+    def __init__(self, bot: OGIROID):
+        self.bot = bot
+
+    @commands.slash_command(name="btc", description="Display current price of BTC")
+    async def help(self, inter):
```
Comment (LevaniVashadze) [inline cogs/btc.py:14]: rename it from help to btc

File: cogs/btc.py [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+        data = response.json()
+        for ticker in data.get('ticker', []):
+            if ticker.get('coin') == 'BTC':
+                btc_prices = ticker.get('value', {})
+        btc_price_usd = btc_prices.get('USD')
+      
+        embbtc = disnake.Embed(
+            title=f"Current BTC Price",
+            description=f"Bitcoin (USD): ${btc_price_usd}",
+            color=disnake.Color.random(),
```
Comment (LevaniVashadze) [inline cogs/btc.py:26]: we use white, so just change random() to .white

**Review by LevaniVashadze — CHANGES_REQUESTED** [review]
Port this command over to Info.py and it's good to go.

**Comment by itsrishub** [comment]
done

**Pull request closed without merging**